src/main/java/redis/clients/jedis/SslHostnameVerifyMode.java

```diff
@@ -0,0 +1,8 @@
+package redis.clients.jedis;
+
+public enum SslHostnameVerifyMode {
```

Comment:
This is the equivalent of the SslVerifyMode in Lettuce, correct?

Reply:
This is exactly SslVerifyMode in Lettuce except the NONE option.
All the remaining options are specific to hostname verification.
So I named this as such.

Reply:
I understand. The `setEndpointIdentificationAlgorithm` uses [standard names](https://docs.oracle.com/javase/8/docs/technotes/guides/security/StandardNames.html) from the Java specification, but I think they are generic and in this case could be misleading if used as part of the API of the driver. A friendly name could describe better (I do not insist on the same naming as in Lettuce, although similar names could make it easier for a app developers to switch between them).

We could always leave a note in the public JavaDoc that this specific enum value would result in using the `HTTPS` Endpoint Identification Algorithm (for verbosity).